singularity-eos/eos/eos_davis.hpp

```diff
@@ -60,6 +60,16 @@ class DavisReactants : public EosBase<DavisReactants> {
       const Real rho, const Real sie, Real *lambda = nullptr) const {
     return Ps(rho) + Gamma(rho) * rho * (sie - Es(rho));
   }
+  PORTABLE_INLINE_FUNCTION Real EntropyFromDensityTemperature(
+      const Real rho, const Real temperature, Real *lambda = nullptr) const {
+    EntropyIsNotEnabled();
+    return 1.0
+  }
+  PORTABLE_INLINE_FUNCTION Real EntropyFromDensityInternalEnergy(
+      const Real rho, const Real sie, Real *lambda = nullptr) const {
+    EntropyIsNotEnabled();
+    return 1.0
+  }
```

Comment:
I like the explicit stubbing out here. :+1:

Reply:
Yeah I thought about trying to do it implicitly, but a) I couldn't figure out a way to do that from the base class while also providing the vector overloads, and b) I figured that it's probably better for every EOS to have the interface entropy functions explicitly defined so that somebody creating a new EOS will understand what to do better.

Reply:
Yep I think that makes sense. :+1: